### DISCUSSION TITLE
integrate getmail gui as plugin

### DISCUSSION THREAD

**Discussion by tonioo** [modoboa/modoboa#4090] [Ideas]
**Originally created by Robert Schetterer on 2013-07-12T06:28:51Z**

Users often have serveral outbound e-mail accouts which the want to collect from,
so integrate a gui for getmail would be very helpfull, perhaps as plugin
http://pyropus.ca/software/getmail/

**Comment by tonioo** [comment]
**Posted by Antoine Nguyen on 2013-07-12T08:36:11Z**

Hi Robert,

getmail seems interesting but I wonder how fetched messages are stored on the server? Into the local user's mailbox?

**Comment by almereyda** [comment]
https://github.com/modoboa/modoboa-imap-migration uses OfflineIMAP and provides basic configurability. It could also suit as a starting point for more fine-grained account migration and GMail like email fetching.

**Comment by tonioo** [comment]
Yes, it could be an interesting feature to add.

**Comment by kryskool** [comment]
Now a [fork](https://github.com/getmail6/getmail6) for Python3 is available

**Comment by kryskool** [comment]
It seem [getmail6 is a hostile fork](https://pyropus.ca./software/getmail/documentation.html#python3-getmail6) said author of original getmail, i don't the same drama like chardet, but i'm not sure if we need to integrate it in Modoboa
